sktime/forecasting/var.py

```diff
@@ -245,7 +245,9 @@ def _predict_interval(self, fh, X, coverage):
 
         for cov in coverage:
             alpha = 1 - cov
-
+            # A hacky way to coerce error-inducing alpha==1 into its approximant
+            if alpha == 1:
```

Comment:
I would do `if alpha >= 0.9999` or similar

Reply:
Sure, I've modified it to `if alpha >= 0.99999`